File: M/2037-minimum-number-of-moves-to-seat-everyone/main.go
package main

import (
	"fmt"
	"sort"
)

func minMovesToSeat(seats []int, students []int) int {
	sort.Ints(seats)
	sort.Ints(students)
	sum := 0
	for i := 0; i < len(seats); i++ {
		d := students[i] - seats[i]
		if d < 0 {
			d = -d
		}
		sum += d
	}
	return sum
}

func main() {
	fmt.Println()
}

File: M/2037-minimum-number-of-moves-to-seat-everyone/main.rs
struct Solution {}

impl Solution {
  pub fn min_moves_to_seat(seats: Vec<i32>, students: Vec<i32>) -> i32 {
    let mut sc: Vec<i32> = seats.clone();
    sc.sort();
    let mut stc: Vec<i32> = students.clone();
    stc.sort();
    (0..sc.len()).map(|i| (sc[i] - stc[i]).abs()).sum()
  }
}
